src/rebar3_typer_prv.erl

```diff
@@ -59,3 +61,21 @@ parse_opts(State) ->
       string => proplists:get_value(string, Args),
       atom => proplists:get_value(atom, Args),
       atom_none => proplists:get_value(atom_none, Args)}.
+
+parse_rebar_config(State) ->
+    Config = rebar_state:get(State, typer, []),
+    lists:foldl(fun parse_rebar_config/2, #{}, proplists:unfold(Config)).
+
+parse_rebar_config({Key, Value}, Opts)
+    when Key == show;
+         Key == show_exported;
+         Key == annotate;
+         Key == annotate_inc_files;
+         Key == edoc;
+         Key == plt;
+         Key == typespec_files;
+         Key == show_success_typings;
+         Key == no_spec ->
+    Opts#{Key => Value};
+parse_rebar_config(Opt, _Opts) ->
+    error({unrecognized_opt, Opt}).
```

Comment:
Can't we do this the rebar3 way? In other words,
```suggestion
    ?PRV_ERROR({unrecognized_opt, Opt}).
```

and then we define a function such as 
```erlang
format_error({unrecognized_opt, Opt}) ->
    io_lib:format("Unrecognized option in rebar.config: ~p", [Opt]);
format_error(...) -> % if there are more clauses
```